src/init.cpp

```diff
@@ -1578,14 +1578,14 @@ bool AppInitMain(NodeContext& node, interfaces::BlockAndHeaderTipInfo* tip_info)
     if (proxyArg != "" && proxyArg != "0") {
         Proxy addrProxy;
         if (IsUnixSocketPath(proxyArg)) {
-            addrProxy = Proxy(proxyArg, proxyRandomize);
+            addrProxy = Proxy(proxyArg, /*tor_stream_isolation=*/proxyRandomize);
```

Comment:
How about change `proxyRandomize` to `tor_stream_isolation` too.

Reply:
i'm not sure. i don't want to go as far as to change interfaces: configuration option name `-proxyrandomize` or the RPC field `proxy_randomize_credential` of `getnetworkinfo`. It would be clearer but is not worth the disruption. And this variable name seems a direct reflection of the option name.